OCDiffTests/OCDAPIComparatorTests.m

```diff
@@ -1171,7 +1189,7 @@ - (void)testAddRemoveForName:(NSString *)name base:(NSString *)base addition:(NS
     XCTAssertEqualObjects(differences, @[], @"Unchanged test failed for %@", name);
 
     // Unchanged, different line number
-    differences = [self differencesBetweenOldSource:addition newSource:[@"\n" stringByAppendingString:addition]];
+    differences = [self differencesBetweenOldSource:addition newSource:[@"" stringByAppendingString:addition]];
```

Comment:
Ahh, this helps explain why the test was failing with this newline. I'll revisit.